geode-core/src/test/java/org/apache/geode/cache/execute/FunctionAdapterJUnitTest.java

```diff
@@ -16,15 +16,26 @@
 
 import static org.junit.Assert.*;
 
+import java.io.DataInputStream;
+import java.io.DataOutputStream;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.Serializable;
+
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 
+import org.apache.geode.internal.InternalDataSerializer;
+import org.apache.geode.internal.Version;
+import org.apache.geode.internal.VersionedDataInputStream;
 import org.apache.geode.test.junit.categories.UnitTest;
+import org.apache.geode.util.test.TestUtil;
 
 @Category(UnitTest.class)
-public class FunctionAdapterJUnitTest {
+public class FunctionAdapterJUnitTest implements Serializable {
```

Comment:
Is the test class only `Serializable` to serialize the anonymous inner class? Seems cleaner to have a static inner class and not make the test class `Serializable`.

Reply:
Good point, I'll make the change